README.md

```diff
@@ -56,6 +56,38 @@ Configuration
   $ sortinghat init <name>
 ```
 
+Migration from "old" MetricsGrimoire identities
+-----------------------------------------------
+
+Assume we had a database named "project_scm", with information about the gi trepository of project "project", with the "old" MetricsGrimoire identities. The process to migrate to Sorting Hat is as follows (assuming sortinghat is already installed):
```

Comment:
`gi trepository` => `git repository`
`Sorting Hat` => `sortinghat` (it is a name)